converters/common.js

```diff
@@ -119,6 +119,31 @@ const TuyaDataTypes = {
     bitmap: 5, // [ 1,2,4 bytes ] as bits
 };
 
+const silvercrestModes = {
+    white: 0,
+    color: 1,
+    scene: 2,
+};
+
+const silvercrestEffects = {
+    steady: '00',
+    snow: '01',
+    rainbow: '02',
+    snake: '03',
+    twinkle: '04',
+    firework: '08',
+    horizontalFlag: '06',
```

Comment:
Keep it snake case, `horizontalFlag` -> `horizontal_flag`

Reply:
Done